src/components/sidebar/left-sidebar/TotalStats.tsx

```diff
@@ -34,7 +35,7 @@ export default function TotalStats() {
           <InformationIcon
             color="#468ac1"
             offset={-12}
-            tooltip="Total number of positive individuals who test positive for antibodies against SARS-CoV-2 (the virus that causes COVID-19) divided by total tests administered, using filtered data"
+            tooltip="Prevalence of antibodies against SARS-CoV-2, aggregated within the filters you have selected.\n\nNB: This should not be interpreted as a representative estimate of seroprevalence in the region of interest, especially if the aggregated studies have biased samples.\n\nPooling technique: fixed effects inverse-variance weighted aggregation of records with sample size over {MIN_DENOMINATOR}."

```

Comment:
@atatmaja - this is one of the tooltips that will need a bit of work

Reply:
did y'all check what the tooltip looks like? if you want to inject the variable you need to use backticks and dollar sign \`string like normal text because we want to showcase: ${variable}\`